eng/tools/BaselineGenerator/BaselineGenerator.csproj

```diff
@@ -5,10 +5,11 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <StartArguments>-o "$(MSBuildThisFileDirectory)../../Baseline.Designer.props"</StartArguments>
     <StartWorkingDirectory>$(MSBuildThisFileDirectory)../../</StartWorkingDirectory>
+    <LangVersion>preview</LangVersion>
```

Comment:
> CSC : error CS1617: Invalid option 'preview' for /langversion. Use '/langversion:?' to list supported values. [D:\a\1\s\eng\tools\BaselineGenerator\BaselineGenerator.csproj]


Which version are you trying to use? I think the latest version supported by the 2.1 SDK is C# 7.3

Reply:
Hmm, I was actually building locally w/ the 2.2.203 SDK. I'll add a `global.json` and see if VS complains about my use of `using var`

Reply:
Either way, the Code Check step is failing w/ `CSC : error CS1617: Invalid option 'preview' for /langversion. Use '/langversion:?' to list supported values. [D:\a\1\s\eng\tools\BaselineGenerator\BaselineGenerator.csproj]`

Reply:
using var is a c# 8 feature, which our 2.x branches don't support. You will have to stick to c# 7